Form-Controls/form-controls.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Could you check this HTML with a [HTML validator](https://validator.w3.org/) and solve any issues?

You can copy & paste your HTML under `Validate by direct input`

Reply:
Perfect score on the validator 🙌